app/views/publish/providers/accredited_providers/edit.html.erb

```diff
@@ -0,0 +1,36 @@
+<% content_for :page_title, title_with_error_prefix("About the accredited provider - Add accredited provider - #{@provider.name_and_code}", @accredited_provider_form.errors.present?) %>
```

Comment:
Should this say 'Edit accredited provider' ?

Reply:
Updated and refactored to match the proto (it doesn't have edit there) 

https://github.com/DFE-Digital/publish-teacher-training/pull/3573/commits/b331083a53b2e9083956e5419f17e3bff3242fee